### PR TITLE
update readme and add clients

### PR DESCRIPTION
**Pull request by anonrig**
Adds multiple clients to README.

A small thank you to @lnear-dev @bungle @alperencantez @TkTech @KusStar for their incredible libraries.

I also invite you to open a pull request to github.com/ada-url/website to add your names and libraries.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,9 +54,13 @@ The Ada library is used by important systems besides Node.js such as Redpanda, K
 
 [![the ada library](http://img.youtube.com/vi/tQ-6OWRDsZg/0.jpg)](https://www.youtube.com/watch?v=tQ-6OWRDsZg)<br />
 
-## Quick Start
+## Installation
+
+Binary packages for the following systems are currently available:
 
+[![Packaging status](https://repology.org/badge/vertical-allrepos/ada.svg)](https://repology.org/project/ada/versions)
```
Comment (lemire) [inline README.md:61]: this is coool!!!

Comment (anonrig) [reply README.md:61]: we should add it to simdutf and simdjson :-)

**Review by lemire — APPROVED** [review]
❤️

**Comment by kassane** [comment]
New client for Dlang (`@safe` & `@nogc` compat based on ada_c header wrapped:
https://github.com/kassane/ada-d

Currently ldc2-1.40 has improved emscripten support and openD-ldc2-1.36 fork (wip: GC support in emscripten, but emalloc not work with typeinfo size) are also compatible.

So wasm32 support will be offered later (ldc2 exclusive)

**Comment by anonrig** [comment]
@kassane looks promising! can you open a PR to add it?

**Pull request merged**